packages/core/src/tools/create-lambda-handler.js

```diff
@@ -203,10 +204,10 @@ const createLambdaHandler = (appRawOrPath) => {
 
           const { skipHttpPatch } = appRaw.flags || {};
           // Adds logging for _all_ kinds of http(s) requests, no matter the library
-          if (!skipHttpPatch) {
+          if (!skipHttpPatch && !event.calledFromCli) {
             const httpPatch = createHttpPatch(event);
-            httpPatch(require('http'));
-            httpPatch(require('https')); // 'https' needs to be patched separately
+            httpPatch(require('http'), logger);
+            httpPatch(require('https'), logger); // 'https' needs to be patched separately
```

Comment:
This ensures HTTP logs and console logs share the same `logger` instance created by `createLogger()`. In the old implementation, there's no difference because every log is a separate request. But in the new implementation, every time you call `createLogger()`, you create a new request, ready to stream logs. Here we want to reuse the same request as much as possible.